examples/installation/CocoaPods.xcodeproj/xcshareddata/xcschemes/App.xcscheme

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Any reason for adding this new schema to the installation examples project

Reply:
Running the installation test locally was sometimes failing due to the App scheme not existing.